staging/src/k8s.io/client-go/util/workqueue/rate_limiting_queue.go

```diff
@@ -82,7 +82,6 @@ func NewRateLimitingQueueWithConfig(rateLimiter RateLimiter, config RateLimiting
 }
 
 // NewNamedRateLimitingQueue constructs a new named workqueue with rateLimited queuing ability.
-// Deprecated: Use NewRateLimitingQueueWithConfig instead.
```

Comment:
these comments seem correct, directing people to the consolidated method is what we want... not sure why we would remove the comments

Reply:
these would cause ~~NewNamedRateLimitingQueue~~ when call NewNamedRateLimitingQueue in ide. And NewNamedRateLimitingQueue impl already change to use NewRateLimitingQueueWithConfig. I you think no need for this, i will close it.